content/S2016/square-event.md

```diff
@@ -0,0 +1,13 @@
+title: Square Tech Workshop: Becoming Ok with Okhttp
+Date: 2016-05-25 
+Category: Events 
+Tags: academic 
+Slug: square-event-s16 
+Author: Bhargavi Dameracharla 
+Summary: Technical workshop on open source HTTP & HTTP/2 client
+
+Square is holding a technical workshop on their open source HTTP & HTTP/2 client for Android and Java applications. 
+Join us to learn more about Okhttp and get a chance to speak with their engineers! Food will be available. 
+(Including vegetarian, vegan, and gluten-free options)
+
+Note: There is an attendance cap of 70 people (50 undergrads and 20 grads), so make sure to register early!
```

Comment:
Also, there is no registration link. Can you add that?

Also, is this event women-only or open to the public?